app.py

```diff
@@ -8,8 +8,13 @@
 
 @app.route("/")
 def index():
-    queries = db.get_queries()
-    return render_template("index.html", queries=queries)
+        searchword = request.args.get('search', '')
+        if len(searchword) > 0:
```

Comment:
Idiomatic way to do things in python is a simple `if searchword` over using `len` to check for the length of the string